### DISCUSSION TITLE
Trying to merge files but obtaining errors in script execution

### DISCUSSION THREAD

**Discussion by dandro180** [xilopaint/alfred-pdf-tools#25] [Q&A] [answered]
Hi, many thanks for the workflow.

I've been trying to merge files but after repeated attempts, it occurred to me to open the debugger view on Alfred, and this is what I see: 

<img width="1394" alt="image" src="https://user-images.githubusercontent.com/79212204/200402414-0db72075-1dcf-4e44-bc89-ad5beeec5cf3.png">

and if I try with the "merge & delete files" option I see this extra bit:

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/79212204/200403019-f03e9c92-1277-407d-a5d5-fb658678a99c.png">

Any way to fix this, please?

**Comment by xilopaint** [comment, marked as answer]
You can install Command Line Tools by running `xcode-select --install` or wait for the next release if you prefer to use the workflow with your Python Homebrew installation.

**Reply by dandro180** [reply]
This solved the problem, thank you very much. Care to explain what's going on? Or a link with more info, please?

**Reply by xilopaint** [reply]
If the workflow worked for you in the past you must have lost your Command Line Tools installation when upgrading to macOS Ventura. In the next release Command Line Tools will no longer be a requirement and the workflow will also work with Homebrew Python.
